testing/testcafe/tests/editors/pages/validationMessagePosition.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
seems like u don't use this page anywhere

Reply:
yeah, forgot this one